Azkfile.js

```diff
@@ -31,4 +31,25 @@ systems({
       BUNDLE_APP_CONFIG: ".bundle",
     },
   },
+  mongodb: {
+    image : { docker: "azukiapp/mongodb" },
```

Comment:
Damn, that shit i did there, that's true, it is only for ruby! :(

I'll fix it, sorry.